pkg/exchange/okex/convert.go

```diff
@@ -164,60 +165,17 @@ func toGlobalTrades(orderDetails []okexapi.OrderDetails) ([]types.Trade, error)
 func toGlobalOrders(orderDetails []okexapi.OrderDetails) ([]types.Order, error) {
 	var orders []types.Order
 	for _, orderDetail := range orderDetails {
-		orderID, err := strconv.ParseInt(orderDetail.OrderID, 10, 64)
-		if err != nil {
-			return orders, err
-		}
-
-		side := types.SideType(strings.ToUpper(string(orderDetail.Side)))
-
-		orderType, err := toGlobalOrderType(orderDetail.OrderType)
-		if err != nil {
-			return orders, err
-		}
-
-		timeInForce := types.TimeInForceGTC
-		switch orderDetail.OrderType {
-		case okexapi.OrderTypeFOK:
-			timeInForce = types.TimeInForceFOK
-		case okexapi.OrderTypeIOC:
-			timeInForce = types.TimeInForceIOC
-
+		isMargin := false
+		if orderDetail.InstrumentType == string(okexapi.InstrumentTypeMARGIN) {
+			isMargin = true
 		}
 
-		orderStatus, err := toGlobalOrderStatus(orderDetail.State)
+		o, err := toGlobalOrder(&orderDetail, isMargin)
 		if err != nil {
-			return orders, err
+			log.WithError(err).Error("order convert error")
```

Comment:
when convert error, you need to "continue"?

Reply:
I return `nil ,err` now. But in MAX convert.go, it still continue while encounter error. So I should fix it too ?

Reply:
you can:

- wrap error with the multierr package and continue to the next order
- or, return the error directly

but we should avoid logging error in the under layer method